README.md

```diff
@@ -98,7 +98,7 @@ var a = 1;
 ```
 
 ## Using your own .jsbeautifyrc options
-The plugin looks for a `.jsbeautifyrc` file in the same directory as the source file you're prettifying (or any directory above if it doesn't exist, or in your home folder if everything else fails) and uses those options along the default ones. [Here](https://github.com/einars/js-beautify/blob/master/js/config/defaults.json)'s an example of how it can look like.
+The plugin looks for a `.jsbeautifyrc` file in the same directory as the source file you're prettifying (or any directory above if it doesn't exist), followed by your home folder, and lastly your personal Sublime settings folder. When one is found, it uses those options along the default ones. [Here](https://github.com/einars/js-beautify/blob/master/js/config/defaults.json)'s an example of how it can look like.
```

Comment:
The order of precedence is: Same directory as source file -> Each parent directory -> $HOME -> Sublime settings dir.

I can reword this if you'd like.